sonar-db/src/main/java/org/sonar/db/property/PropertiesDao.java

```diff
@@ -162,6 +162,10 @@ public PropertyDto selectProjectProperty(DbSession dbSession, long componentId,
       partitionComponentIds -> getMapper(session).selectByKeysAndComponentIds(partitionKeys, partitionComponentIds)));
   }
 
+  public List<PropertyDto> selectPropertiesByComponentIds(DbSession session, Set<Long> componentIds) {
+    return executeLargeInputs(componentIds, partitionIds -> getMapper(session).selectByComponentIds(partitionIds));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1612)